container.go

```diff
@@ -15,14 +15,13 @@ type Container struct {
 }
 
 // NewContainer returns a new Container instance holding the specified CanvasObjects.
-//
```

Comment:
This bank line is required.
Also please add one to the other deprecated notices - it needs to begin a new paragraph.